### PR TITLE
Added the EN version of the Benchmark reports

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the benchmark reports (Sysbench and TPC-C) for v5.0 GA.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5904
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/benchmark-sysbench-v5-vs-v4.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+title: TiDB Sysbench Performance Test Report - v5.0 vs. v4.0
+---
+
+# TiDB Sysbench Performance Test Report -- v5.0 vs. v4.0
+
+## Test purpose
+
+This test aims at comparing the Sysbench performance of TiDB 5.0 and TiDB 4.0 in the Online Transactional Processing (OLTP) scenario.
```
Comment (TomShawn) [inline benchmark/benchmark-sysbench-v5-vs-v4.md:9]:
```suggestion
This test aims at comparing the Sysbench performance of TiDB v5.0 and TiDB v4.0 in the Online Transactional Processing (OLTP) scenario.
```

File: benchmark/v5.0-performance-benchmarking-with-tpcc.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+---
+title: TiDB TPC-C Performance Test Report -- v5.0 vs. v4.0
+---
+
+# TiDB TPC-C Performance Test Report -- v5.0 vs. v4.0
+
+## Test purpose
+
+This test aims at comparing the TPC-C performance of TiDB 5.0 and TiDB 4.0 in the Online Transactional Processing (OLTP) scenario.
```
Comment (TomShawn) [inline benchmark/v5.0-performance-benchmarking-with-tpcc.md:9]:
```suggestion
This test aims at comparing the TPC-C performance of TiDB v5.0 and TiDB v4.0 in the Online Transactional Processing (OLTP) scenario.
```

**Comment by TomShawn** [comment]
@qiancai Please resolve the conflict by aligning the TOC with
image

**Review by shuke987 — COMMENTED** [review]
LGTM

**Pull request merged**